assets/js/admin-scripts.js

```diff
@@ -61,6 +56,20 @@ jQuery(document).ready(function ($) {
 
 			});
 
+			/** manually update any select box values */
+			clone.find( 'select' ).each(function() {
+				$( this ).val( row.find( 'select[name="' + $( this ).attr( 'name' ) + '"]' ).val() );
+			});
+
+			/** manually uncheck any checkboxes */
+			clone.find( 'input[type="checkbox"]' ).each(function() {
+				var checked = $(this).is(':checked');
+				if ( checked ) {
+					$(this).prop('checked', false);
+				}
+				$(this).val(1);
```

Comment:
@SDavisMedia Let's drop a comment before this line detailing why we are setting the value (due to the foreach above).

Should just make less confusion if someone looks and sees the unset above, and then us setting it here. Other than that, looks/works great! Nice job.